### PR TITLE
Feature/aqi 4241

### PR DESCRIPTION
**Pull request by dedwards-AQI**

### CODE REVIEW

**Review by KentMiller — DISMISSED** [review]

**Review by AlanLiu-AI — DISMISSED** [review]
Lgtm.

File: pyproject.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@ requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
 [project]
 name = "one_py_sdk"
-version = "1.2.0"
+version = "1.3.3"
```
Comment (JonathanRobertson-AQI) [inline pyproject.toml]: If you're just adding new functionality, shouldn't this change from 1.2.0 to 1.3.0?

**Review by JonathanRobertson-AQI — DISMISSED** [review]

File: src/one_py_sdk/enterprise/report.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+from one_py_sdk.enterprise.authentication import AuthenticationApi
+from one_py_sdk.shared.helpers.protobufhelper import DeserializeResponse
+
+class ReportApi:
+    def __init__(self, env, auth: AuthenticationApi):
+        self.AppUrl = "/enterprise/report/v1/"
+        self.Environment = env
+        self.Authentication = auth
+        
+    def GetReportDefinitions(self, plantId=None):
```
Comment (JonathanRobertson-AQI) [inline src/one_py_sdk/enterprise/report.py:15]: do we want to use plantId or operationId?

Comment (dedwards-AQI) [reply src/one_py_sdk/enterprise/report.py:15]: Everywhere else in the code it is plantId. I could go and change it everywhere, but for now I thought I'd just keep it consistent.

File: pyproject.toml (hunk 1 above)
Comment (dedwards-AQI) [reply pyproject.toml]: I'll bump the version to 1.4.0. There's already a 1.3.2 out there, and somehow the version bump got lost in the shuffle and didn't get merged to master.

File: src/one_py_sdk/enterprise/report.py (hunk 2 above)
Comment (JonathanRobertson-AQI) [reply src/one_py_sdk/enterprise/report.py:15]: I like consistency.  If the SDK consumer is already using PlantID, I'd keep it that way.  Thanks!

**Pull request merged**